webapp/Sigbuilder-graph.js

```diff
@@ -0,0 +1,1872 @@
+    var max_x = sigbuilder_Graph.xAxis[0].getExtremes().max;
+    var min_y = sigbuilder_Graph.yAxis[0].getExtremes().min;
+    var max_y = sigbuilder_Graph.yAxis[0].getExtremes().max;
+    var labelArray = ['xmin','xmax','ymin','ymax'];
+    var textValueArray = [min_x,max_x,min_y,max_y];
+    for(var i = 0; i < labelArray.length; i++){
+        // Input Title
+        var namelabel = document.createElement('label');
+        namelabel.innerHTML = labelArray[i];
+        namelabel.style.marginLeft = "30px";
+        myform.appendChild(namelabel);
+
+        var value = 0;
+        if(((textValueArray[i]).toString()).includes(".")){
+            value = (textValueArray[i]).toFixed(6);
+        }else{
+            value = (textValueArray[i]);
+        }
+        // Input
+        var input = document.createElement("input");
+        input.name = "edit_"+labelArray[i].toString();
+        input.value = value;
+        input.setAttribute("id", "edit_"+labelArray[i].toString());
+        input.setAttribute("class", "fieldInput");
+        myform.appendChild(input);
+
+        // Line break
+        var linebreak = document.createElement('br');
+        myform.appendChild(linebreak);
+
+        // Line break
+        var linebreak = document.createElement('br');
+        myform.appendChild(linebreak);
+
+    }
+    // Line break
+    var linebreak = document.createElement('br');
+    myform.appendChild(linebreak);
+
+    // Button - Cancel
+    var cancel_btn = document.createElement("button");
+    cancel_btn.style.cssFloat = "right";
+    cancel_btn.innerHTML = 'Cancel';
+    cancel_btn.type = "button";
+    cancel_btn.name = "Cancel";
+    myform.appendChild(cancel_btn);
+
+    // Button - OK
+    var ok_btn = document.createElement("button");
+    ok_btn.style.cssFloat = "right";
+    ok_btn.style.marginRight = "20px";
+    ok_btn.innerHTML = 'OK';
+    ok_btn.type = "button";
+    ok_btn.name = "OK";
+
+    myform.appendChild(ok_btn);
+    content.appendChild(myform);
+    var height = 200;
+    var wind = showModalWindow(graph, 'Scilab Multiple Values Request', content, 300, height);
+
+    // Executes when button 'cancel_btn' is clicked
+    cancel_btn.onclick = function() {
+        document.getElementById("messageLabel").innerHTML = "";
+        wind.destroy();
+    };
+    // Executes when button 'ok_btn' is clicked
+    ok_btn.onclick = function() {
+        var x_min = parseFloat(document.getElementById("edit_xmin").value);
+        var x_max = parseFloat(document.getElementById("edit_xmax").value);
+        var y_min = parseFloat(document.getElementById("edit_ymin").value);
+        var y_max = parseFloat(document.getElementById("edit_ymax").value);
+        if(x_min < 0){
+            document.getElementById("messageLabel").innerHTML = "X should be positive";
+        }else{
+            document.getElementById("messageLabel").innerHTML = "";
+        }
+        sigbuilder_Graph.xAxis[0].update({
+                max: parseFloat(x_max)
```

Comment:
```suggestion
                max: x_max
```

parseFloat already called above